docs/docs/recipes/manage-users/on-admin-console-page.md

```diff
@@ -7,7 +7,18 @@ sidebar_position: 1
 
 ## Add user
 
-TODO: admin manually add
+Not only can the end-user sign up by his/herself, but also the admin users can manually add accounts for them.
+
+1. In the "User Management" panel, click the "+ Add User" button at the top right corner.
```

Comment:
nit: feels like, `on the pane` is more comfortable, not sure, just a feeling.

Reply:
The user may view the the “Add user” section directly, and don't know which "panel".
So I emphasize the "User Management" panel here.

Later we may reorg this step like #46 

<img width="732" alt="image" src="https://user-images.githubusercontent.com/10594507/175454252-d918686a-291e-4470-bf20-45ef7a1b9b93.png">